meerk40t/gui/toolwidgets/toolparameter.py

```diff
@@ -0,0 +1,528 @@
+import math
+
+import wx
+
+from meerk40t.gui.scene.sceneconst import RESPONSE_CHAIN, RESPONSE_CONSUME
+from meerk40t.gui.toolwidgets.toolwidget import ToolWidget
+
+_ = wx.GetTranslation
+
+
+class SimpleSlider:
```

Comment:
This belongs in it's own more general set of scene objects to add to a scene as a generalized thing. Fine for now.